source/_patterns/02-molecules/vue-widgets/src/vue-cards/components/banner.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div 
+    class="vue-banner"
+    :style="styles"
+    @click="randomColor()"
+  >
+    <h3><marquee>{{ message }}</marquee></h3>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Banner',
+  data() {
+    return {
+      styles: {
+        'background-color': 'black',
+      },
+      message: "Good news everyone! Store's open for business.",
+    };
+  },
+  created() {
+    setInterval(() => {
+      this.randomColor();
+    }, 3000);
+  },
+  methods: {
+    randomColor() {
+      const rand255 = () => Math.round(Math.random() * 255);
+      this.styles[
+        'background-color'
+      ] = `rgba(${rand255()},${rand255()},${rand255()},${1})`;
+    },
+  },
+};
+</script>
+
+<style lang="scss">
+// importing NON PRINTING styles in sass
+@import '00-protons/variables';
+
+.vue-banner {
```

Comment:
Get me a CSS3 transition on background color so it's super smooth on the transition.